inginious/frontend/parsable_text.py

```diff
@@ -122,7 +122,7 @@ def starttag(self, node, tagname, suffix='\n', empty=False, **attributes):
 class ParsableText(object):
     """Allow to parse a string with different parsers"""
 
-    def __init__(self, content, mode="rst", show_everything=False, translation=gettext.NullTranslations()):
+    def __init__(self, content, mode="rst", show_everything=False, translation=gettext.NullTranslations(), debug=False):
```

Comment:
I would rather define debug inside `parse(debug=False)`, and call it manually.

Reply:
done